src/components/Pages/RequestPage/RequestPage.tsx

```diff
@@ -60,7 +60,7 @@ export const RequestPage = () => {
   const [isTransactionModalOpen, setIsTransactionModalOpen] = useState(false)
   // TODO: Add a type for the request.
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const requestRef = useRef<any>()
+  const requestRef = useRef<{ method: string; params: any[]; code?: number }>()
```

Comment:
can we avoid that any? and remove old comment?

Reply:
I need to review the `auth-server` code to get the correct return type. As there's a log in issue right, now, it might be better to merge it as it and then correct the types, what do you think?